packages/oauth/src/install-provider.ts

```diff
@@ -317,8 +318,9 @@ export class InstallProvider {
    * Handles the install path (the default is /slack/install) requests from an app installer.
   */
   public async handleInstallPath(
-    _req: IncomingMessage,
+    req: IncomingMessage,
     res: ServerResponse,
+    options?: InstallPathOptions,
```

Comment:
Small thing and this is not a hard requirement but maybe we can rename the two options arguments here? Perhaps more applicable is `pathOptions` and `urlOptions`?

Reply:
@filmaj The reason why I chose this name is for the consistency with handleCallback code. Renaming `options` in these two methods may be fine but we can make changes later (as this is just internal naming topic)

Reply:
I'm fine with whatever you decide!